src/components/Navigation/Navigation.js

```diff
@@ -23,7 +23,7 @@ export default function Navigation() {
 			<Link href="./breathwork">
 				<StyledLink active={router.pathname === '/breathwork'}>
 					<SVG variant="lotus" />
-					Breath deeply
+					Breathe deeply
```

Comment:
Major mistakes like this can't be approved. Sawray~

Reply:
Sind beide soweit ich weiß korrekt